src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -427,11 +408,6 @@ else if (conf.disk_access_mode == Config.DiskAccessMode.mmap_index_only)
             logger.info("DiskAccessMode is {}, indexAccessMode is {}", conf.disk_access_mode, indexAccessMode);
         }
 
-        if (conf.gc_warn_threshold_in_ms < 0)
-        {
-            throw new ConfigurationException("gc_warn_threshold_in_ms must be a positive integer");
```

Comment:
Are we still throwing a `ConfigurationException` if `gc_warn_threshold_in_ms` is negative? I'm kind of losing the thread in `PropertiesChecker`...

Reply:
Which thread you mean? I am not sure

Reply:
Not a literal thread...just trying to figure out if `we still throw a ConfigurationException if gc_warn_threshold_in_ms is negative`

Reply:
I don't think it can be set negative anymore with the new `CassandraDuration`